pyproject.toml

```diff
@@ -24,21 +24,22 @@ dependencies = [
     "pandas>=2.2.3",
     "pydantic>=2.11.4",
     "pytest>=8.3.5",
-    "python-decouple>=3.8",
-    "openapi-spec-validator>=0.7.2",
-    "jsonref>=1.1.0",
     "pydantic-ai>=0.4.2",
-    "yarl>=1.20.1",
     "rich>=14.0.0",
     "sqlparse>=0.5.3",
     "tabulate>=0.9.0",
     "ibis-framework>=10.6.0",
     "pyarrow<19.0.0",
     "executing>=2.2.0",
+    "fastapi>=0.116.1",
+    "duckdb>=1.3.2",
+    "fsspec>=2025.7.0",
+    "markitdown>=0.1.3",
```

Comment:
I think this pulls in all of the bloatware that comes with markitdown again. Maybe we can limit the dependencies like before: `"markitdown[docx,pdf,pptx,xls,xlsx]>=0.1.2"`?